testcases/OpTestKernelTest.py

```diff
@@ -109,8 +106,8 @@ def is_url(path):
         log.info("Upstream kernel commit-time: %s", tcommit)
         log.debug("Compile the upstream kernel")
         try:
-            cpu= self.con.run_command("lscpu | grep '^CPU(s):' | awk '{print $2}'")
-            err=self.con.run_command("make -j {} -s vmlinux".format(cpu[-1]), timeout=self.host_cmd_timeout)
+            cpu = int(self.con.run_command("lscpu --online -e|wc -l")[-1])
```

Comment:
Please push this  a utility function that can be reused in other places like example Gcov , kernel upstream build test